packages/graphback-codegen-schema/src/definitions/schemaDefinitions.ts

```diff
@@ -292,9 +292,9 @@ function mapObjectInputFields(schemaComposer: SchemaComposer<any>, fields: Graph
 
     let inputType
     if (isObjectType(namedType)) {
-      if (isModelType(namedType)) {
-        throw new Error(`Non-model ${objectName}.${field.name} maps to Graphback model (${namedType.name}) which is not allowed.`)
-      }
+      // if (isModelType(namedType)) {
+      //   throw new Error(`Non-model ${objectName}.${field.name} maps to Graphback model (${namedType.name}) which is not allowed.`)
+      // }
```

Comment:
Should we remove the comment code all together?

Reply:
Removed, thanks.